src/molgenis/capice/cli/args_handler_explain.py

```diff
@@ -0,0 +1,96 @@
+import pickle
+
+from molgenis.capice import __version__
+from molgenis.capice.main_explain import CapiceExplain
+from molgenis.capice.core.capice_manager import CapiceManager
+from molgenis.capice.utilities.input_processor import InputProcessor
+from molgenis.capice.cli.args_handler_parent import ArgsHandlerParent
+from molgenis.capice.validators.model_validator import ModelValidator
+from molgenis.capice.validators.version_validator import VersionValidator
+
+
+class ArgsHandlerExplain(ArgsHandlerParent):
+    """
+    Handler for the CAPICE submodule Explain
+    """
+
+    def __init__(self, parser):
+        super(ArgsHandlerExplain, self).__init__(parser=parser)
+
+    @property
+    def _extension(self):
+        return '.tsv', '.tsv.gz'
+
+    @property
+    def _required_output_extensions(self):
+        return '.tsv', '.tsv.gz'
+
+    @property
+    def _empty_output_extension(self):
+        return self._required_output_extensions[1]
+
+    def create(self):
+        self.parser.add_argument(
+            '-i',
+            '--input',
+            action='append',
+            type=str,
+            required=True,
+            help='path to trained model (.dat)'
+        )
+        self.parser.add_argument(
+            '-o',
+            '--output',
+            action='append',
+            type=str,
+            help='path to directory or filename (or both) for export. '
+                 'Has to be in .tsv or .tsv.gz extension. '
+                 'Will always be gzipped!'
+        )
+        self.parser.add_argument(
+            '-f',
```

Comment:
A stack trace is presented instead of just the error message:

Actual:
```
python capice.py explain -i v3.0.0-v1_grch38.pickle.dat -o grch38.tsv.gz
python capice.py explain -i v3.0.0-v1_grch38.pickle.dat -o grch38.tsv.gz
Traceback (most recent call last):
  File "capice\src\molgenis\capice\capice.py", line 16, in <module>
    main()
  File "capice\src\molgenis\capice\capice.py", line 12, in main
    argument_handler.handle()
  File "capice\src\molgenis\capice\core\args_handler.py", line 34, in handle
    args.func(args)
  File "capice\src\molgenis\capice\cli\args_handler_explain.py", line 74, in _handle_args
    input_processor = InputProcessor(
  File "capice\src\molgenis\capice\utilities\input_processor.py", line 34, in __init__
    self._handle_input_output_directories()
  File "capice\src\molgenis\capice\utilities\input_processor.py", line 60, in _handle_input_output_directories
    self._check_force()
  File "capice\src\molgenis\capice\utilities\input_processor.py", line 65, in _check_force
    raise FileExistsError(
FileExistsError: Output file grch38.tsv.gz already exists! Use -f / --force to overwrite.
```

Expected something like:
```
python capice.py explain -i v3.0.0-v1_grch38.pickle.dat -o grch38.tsv.gz
python capice.py explain -i v3.0.0-v1_grch38.pickle.dat -o grch38.tsv.gz
FileExistsError: Output file grch38.tsv.gz already exists! Use -f / --force to overwrite.
```